codegen/gateway.go

```diff
@@ -578,6 +577,35 @@ func testFixtures(endpointConfigObj map[string]interface{}) (map[string]*Endpoin
 	return ret, err
 }
 
+func loadHeadersFromConfig(endpointConfigObj map[string]interface{}, key string) ([]string, map[string]string, error) {
+	headers, ok := endpointConfigObj[key]
+	if !ok {
+		return nil, nil, errors.Errorf("Unable to parse %s", key)
```

Comment:
nit: the convention is have the first letter in lower case in the error message so that when the error is wrapped the message is like, "error blah blah: unable to parse ..." instead of "error blah blah: Unable to parse ...". Also "%q" is arguably a better formatter as it makes the key stand out.

Reply:
noted, I addressed this style in https://github.com/uber/zanzibar/pull/366

also added TODO to capture `endpointConfigObj` cast as interface issue there